lib/matplotlib/tests/test_table.py

```diff
@@ -79,3 +79,25 @@ def test_label_colours():
               colColours=colours,
               colLabels=['Header'] * dim,
               loc='best')
+
+
+@image_comparison(baseline_images=['table_cell_manipulation'],
+                  extensions=['png'])
+def test_diff_cell_table():
+    l1 = ('', 't0', 't1', 't2', 't3 ', 't4', 't5', 't6')
+    l2 = ('DLI', 35, 38, 10, 22, 25, 85, 22)
+    cells = ('horizontal', 'vertical', 'open', 'closed', 'T', 'R', 'B', 'L')
+
+    fig = plt.figure(figsize=(4, len(cells)+1))
+    for i, cell in enumerate(cells, 1):
+        ax = fig.add_subplot(8, 1, i)
+        ax.plot([1, 2, 3, 4, 5, 6])
```

Comment:
What is the rationale for including this blue line?  If it is not essential, better to leave it out.